scripts/resolve_suggestions_urls.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import json
+import os
+from jsonpath_rw import parse
+from jsonpointer import set_pointer
+
+SUGGESTIONS_API_URL = (
```

Comment:
The naming here seems a bit strange, given we aren't calling an api. Something like `unformatted_suggestions_url_root` might make more sense.

Reply:
Renamed, thanks!👍